apps/costcontrol/js/config/config_manager.js

```diff
@@ -53,7 +53,8 @@ var ConfigManager = (function() {
     'resetTime': 1,
     'trackingPeriod': 'monthly',
     'isMobileChartVisible': true,
-    'isWifiChartVisible': false
+    'isWifiChartVisible': false,
+    'supportCustomizeMode': false
```

Comment:
Instead of making `supportCustomizeMode` part of the async options of the program. Let's add it simply as a property of `ConfigManager`.